server/py/framework/db/sqldb/db.py

```diff
@@ -5995,6 +5994,78 @@ def list_alerts(
             alerts.append(alert)
         return alerts
 
+    def delete_project_alerts(
```

Comment:
```suggestion
    def delete_alerts_and_notifications(
```
Because you are also deleting the notifications

Reply:
@moranbental technically, I delete everything related to alerts here - configs, notifications. So, I will leave it as is